tests/smtp-raw-extraction/suricata.yaml

```diff
@@ -0,0 +1,54 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-4.0.3.yaml
```

Comment:
this should not be necessary (in fact I think we should remove it from the instructions)